### ISSUE TITLE
[dev] use svg for vectors and diagrams

### ISSUE THREAD

**Issue by pi0** [nuxt/nuxtjs.org#397]
Checking [guides](https://github.com/nuxt/nuxtjs.org/tree/dev/static/guides) directory, we are currently using jpeg and png for diagrams. Jpeg compression is always lossy and for png files are also using lossy compression format which decreases text readability and makes size bigger. I would recommend using SVG format as even internet explorer supports it.

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxtjs.org/issues/c227">#c227</a>)</em></sub></div>

**Comment by Krutie** [comment]
@pi0 Having images in an SVG format will be good. I'm able to export SVG (440KB) from LucidChart and then run it through SVGOMG to bring it little bit down to ~330KB. I can provide images in SVG format in future PRs.

**Comment by debs-obrien** [comment]
agree, svgs would be much better. is it possible to get the previous images in svg format @Krutie. Not urgent but whenever you have the time.

**Comment by Krutie** [comment]
@debs-obrien Yes, certainly. I will send PR with SVG versions of context diagram (after implementing @pi0's changes) and Nuxt lifecycle image.

**Comment by debs-obrien** [comment]
Hi @Krutie what is the status of nuxt lifecycle image in svg format? it would also be great to get the modules one too. And how is the raw formats coming along? leet us know if it takes too long to do this or at least some feedback and if there is anything we can help on it.

**Comment by Krutie** [comment]
Hey @debs-obrien Modules diagram was done on issue #400. We're kind of waiting on Pooya's review.

As soon as I get a chance, I will get the Nuxt lifecycle diagram done.

**ToDo**
- [ ] Convert Nuxt Lifecycle Diagram into .drawio format

**Comment by debs-obrien** [comment]
Great, thanks @Krutie
